index.js

```diff
@@ -47,11 +48,22 @@ class Conf {
 		};
 
 		if (!options.cwd) {
```

Comment:
I've already commented it below, but if `options.cwd` is defined, the logic of getting the default `options.projectVersion` will never be run.  Worth adding a test for this so that you see that it's an issue with the current implementation.

Reply:
You're right, updated it and the tests!

Can you check again? :)